package.json

```diff
@@ -1,32 +1,28 @@
 {
-  "name": "ExpandingTextareas",
-  "version": "0.2.0",
-  "description": "jQuery plugin for elegant expanding textareas",
+  "name": "expanding-textareas",
+  "version": "1.0.0",
+  "description": "An elegant approach to making textareas automatically grow.",
   "homepage": "http://bgrins.github.com/ExpandingTextareas/",
   "repository": {
     "type": "git",
     "url": "git://github.com/bgrins/ExpandingTextareas.git"
   },
   "bugs": "https://github.com/bgrins/ExpandingTextareas/issues",
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "http://mit-license.org/"
-    }
-  ],
+  "license": "MIT",
   "author": {
     "name": "Brian Grinstead",
     "web": "http://briangrinstead.com/"
   },
-  "main": "./expanding.js",
+  "main": "dist/expanding.js",
```

Comment:
How should we distribute this both as a jQuery plugin and non?  Basically the same comment as in the bower config file, but should we be pointing to `dist/expanding.jquery.js` instead, or in addition to this somehow?